settings/deno.vim

```diff
@@ -13,7 +13,11 @@ augroup vim_lsp_settings_deno
       \     'implementations': v:true,
       \     'references': v:true,
       \     'referencesAllFunctions': v:true,
+      \     'test': v:true,
+      \     'testArgs': ['--allow-all'],
```

Comment:
[memo]
specify default value.
both default values are `"test: true"` and `"testArgs: ["--allow-all"]`